frontends/bmd/src/pages/accessible_controller_diagnostic_screen.tsx

```diff
@@ -0,0 +1,288 @@
+import React, { useEffect, useState } from 'react';
+import { Button, Main, MainChild, Prose, Text } from '@votingworks/ui';
+import { DateTime } from 'luxon';
+import styled from 'styled-components';
+import { Screen } from '../components/screen';
+import { ScreenReader } from '../config/types';
+
+type ButtonName = 'Up' | 'Down' | 'Left' | 'Right' | 'Select';
+
+const Header = styled(Prose).attrs({
+  maxWidth: false,
+})`
+  display: flex;
+  align-items: baseline;
+  justify-content: space-between;
+`;
+
+const StepContainer = styled.div`
+  display: grid;
+  grid-template-columns: 55% 1fr;
+  flex-grow: 1;
+  align-items: center;
+  padding: 0 6em;
+
+  svg {
+    justify-self: flex-end;
+    height: 25em;
+  }
+
+  button {
+    margin-top: 4em;
+  }
+
+  ol {
+    margin-top: 0;
+    padding-left: 1em;
+    li {
+      margin-bottom: 1em;
+    }
+  }
+`;
+
+interface AccessibleControllerIllustrationProps {
+  highlight?: ButtonName | 'Headphones';
+}
+
+function AccessibleControllerIllustration({
```

Comment:
maybe make this its own file? Are we able to make an svg file or will that not work because of the parameters in the svg?

Reply:
I first tried making an SVG file but couldn't figure out how to parameterize it without adding a complicated step to the build (there's a tool called SVGR that automatically turns an SVG file into a React component, but no basic TS way to import an SVG as tags, only as an image). I can move it to a separate file

Reply:
Actually now that I try moving it, I'm realizing it relies on the ButtonName type from this module, so I think it's actually just easier to leave it in the same file for now.